libs/thread_executors/tests/unit/resource_manager.cpp

```diff
@@ -193,7 +193,8 @@ void test_executors_shrink(std::size_t num_pus)
     using namespace hpx::parallel::execution;
     std::size_t processing_units = hpx::get_os_thread_count();
 
-    processing_units = (processing_units / num_pus) * num_pus;
+    processing_units = (std::min)(
+        ((processing_units + 1) / num_pus) * num_pus - 1, std::size_t(1));
```

Comment:
Why the `min`? Looks like that makes `processing_units` either 0 or 1, always. I don't quite understand what the test is checking so that may be correct...

Reply:
Right, this is wrong. I'll go back and fix this.